apps/game/src/edit/roads.rs

```diff
@@ -1013,11 +1013,19 @@ fn fade_irrelevant(app: &App, r: RoadID) -> GeomBatch {
     }
 
     // The convex hull illuminates a bit more of the surrounding area, looks better
-    let fade_area = Polygon::with_holes(
-        map.get_boundary_polygon().clone().into_ring(),
-        vec![Polygon::convex_hull(holes).into_ring()],
-    );
-    GeomBatch::from(vec![(app.cs.fade_map_dark, fade_area)])
+    match Polygon::convex_hull(holes) {
+        Ok(hole) => {
+            let fade_area = Polygon::with_holes(
+                map.get_boundary_polygon().clone().into_ring(),
+                vec![hole.into_ring()],
+            );
+            GeomBatch::from(vec![(app.cs.fade_map_dark, fade_area)])
+        }
+        Err(_) => {
```

Comment:
![Screenshot from 2022-08-30 10-15-59](https://user-images.githubusercontent.com/1664407/187399080-e0639c17-be5b-4af9-a447-1d436437f775.png)
As a reminder, this is the "fade irrelevant" effect. In my manual tests so far, it all still works everywhere. The fallback of not fading stuff seems fine.